packages/remix-cloudflare-pages/worker.ts

```diff
@@ -0,0 +1,96 @@
+import type { ServerBuild, AppLoadContext } from "@remix-run/server-runtime";
+import { createRequestHandler as createRemixRequestHandler } from "@remix-run/server-runtime";
+
+export interface CreateRequestHandlerParams<Env = any> {
+  build: ServerBuild;
+  getLoadContext?: (context: EventContext<Env, any, any>) => AppLoadContext;
+  mode?: string;
+}
+
+function createInternalHandler<Env = any>({
+  build,
+  getLoadContext,
+  mode
+}: CreateRequestHandlerParams<Env>): PagesFunction<Env> {
+  let platform = {};
+  let handleRequest = createRemixRequestHandler(build, platform, mode);
+
+  return context => {
+    let loadContext =
+      typeof getLoadContext === "function"
+        ? getLoadContext(context)
+        : undefined;
+    return handleRequest(context.request, loadContext);
+  };
+}
+
+type EnvWithAssets = Record<string, unknown> & {
+  ASSETS: { fetch: typeof fetch };
+};
+
+async function handleAsset(request: Request, env: unknown) {
```

Comment:
The `@remix-run/cloudflare-workers` package exports this asset handler and the request handler function. The docs show a 'simplified' version which uses the primary export and a 'low-level' version where developers can customize the entrypoint more themselves. Are you moving away from that 'low-level' option by not exporting these same functions here?

Reply:
There isn't a separate package required to serve assets anymore and it's a simple couple liner so I'm not going to export this. We will still document the lower level handler pattern.